### PR TITLE
ASM-4531 Prefer env var password for disco scripts

### PR DESCRIPTION
**Pull request by logankimmel**

### CODE REVIEW

**Comment by jkrauskopf** [comment]
@ripienaar or @sushilrai Please review, thanks.

**Comment by ripienaar** [comment]
@logankimmel i think we're ditching this way of passing the passwords and just moving to ```ENV["PASSWORD"]``` right? based on discussions on asm-deployer?

So don't want to merge this if we are

File: bin/discovery.rb [hunk 1]
```diff
@@ -20,6 +20,8 @@
   opt :community_string, 'dummy value for ASM, not used'
 end
 
+opts[:password] = ENV["#{opts[:server]}-pass"] || opts[:password]
```
Comment (gavin-scott) [inline bin/discovery.rb]: This needs to be updated to match https://github.com/dell-asm/dell-equallogic/pull/43

**Comment by logankimmel** [comment]
@gavin-scott updated

**Pull request merged**